specification/hello-world.yaml

```diff
@@ -20,7 +20,7 @@ info:
     email: api.management@nhs.net
 
 servers:
-  - url: https://emea-demo8-test.apigee.net
+  - url: https://internal-dev.api.service.nhs.uk
```

Comment:
Again check with Tony what he wants here - I suspect `sandbox.api.service.nhs.uk` is correct here

Reply:
@Sourceless you're right